service-monitor/src/main/java/com/yahoo/vespa/service/duper/DuperModelManager.java

```diff
@@ -48,14 +50,20 @@
     public DuperModelManager(ConfigserverConfig configServerConfig, FlagSource flagSource, SuperModelProvider superModelProvider) {
         this(configServerConfig.multitenant(),
                 configServerConfig.serverNodeType() == ConfigserverConfig.ServerNodeType.Enum.controller,
-                superModelProvider, new DuperModel(), flagSource);
+             superModelProvider, new DuperModel(), flagSource, SystemName.from(configServerConfig.system()));
     }
 
     /** For testing */
-    DuperModelManager(boolean multitenant, boolean isController, SuperModelProvider superModelProvider, DuperModel duperModel, FlagSource flagSource) {
+    DuperModelManager(boolean multitenant, boolean isController, SuperModelProvider superModelProvider, DuperModel duperModel, FlagSource flagSource, SystemName system) {
         this.duperModel = duperModel;
 
-        if (multitenant) {
+        if (system == SystemName.dev) {
+            supportedInfraApplications =
```

Comment:
Remove `isController` check and add a TODO for including `controllerApplication`.